tests/common/runners/test_base_runner.py

```diff
@@ -178,6 +178,45 @@ def test_filter_ignored_directories_regex_absolute_cwd(self):
         # we expect .terraform and dir11/dir2 and dir33/dir2 to get filtered out
         self.assertEqual(set(remaining_dirs), expected)
 
+    def test_filter_ignored_directories_by_values(self):
+        # this simulates scanning a subdirectory and applying filter logic using an absolute path
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+        excluded_paths = ['dir2++']
```

Comment:
can you use dirname with a meaningfull name for the test?
I assume you are trying to use something that does not compile to a regex, so you can call it `not_regex_dir++` or something like that